ocs_ci/utility/utils.py

```diff
@@ -2194,10 +2194,28 @@ def get_csi_versions():
     # importing here to avoid circular imports
     from ocs_ci.ocs.ocp import OCP
 
-    for provisioner in [
-        constants.CSI_CEPHFSPLUGIN_PROVISIONER_LABEL,
-        constants.CSI_RBDPLUGIN_PROVISIONER_LABEL,
-    ]:
+    hci_platform_conf = (
+        config.ENV_DATA["platform"].lower() in constants.HCI_PROVIDER_CLIENT_PLATFORMS
+    )
+    # csi provisioner pods were renamed starting from 4.18 for Provider mode and 4.19 for every mode (Converged mode)
+    if (
+        version_module.get_semantic_ocs_version_from_config()
+        >= version_module.VERSION_4_18
+        and hci_platform_conf
+        or version_module.get_semantic_ocs_version_from_config()
+        >= version_module.VERSION_4_19
+    ):
+        provisioner_list = [
+            constants.CSI_CEPHFSPLUGIN_PROVISIONER_LABEL_419,
```

Comment:
There are two constants in constants.py with the same value.

Reply:
you're right. I am using the constant from Vijays recent PR https://github.com/red-hat-storage/ocs-ci/commit/867e4e36088009671efd5d25919ca8ad62a13614. 
The var name `CSI_CEPHFSPLUGIN_PROVISIONER_LABEL_419`  describes the value.

Reply:
Okay, in a different PR we can remove the duplicate after changing the usages.